common/djangoapps/student/models.py

```diff
@@ -930,6 +930,34 @@ def clear_lockout_counter(cls, user):
         except ObjectDoesNotExist:
             return
 
+    def __repr__(self):
+        """Repr -> LoginFailures(username, count, date)"""
+        date_str = '-'
+        if self.lockout_until is not None:
+            date_str = self.lockout_until.isoformat()
+
+        return 'LoginFailures({username}, {count}, {date})'.format(
+            username=self.user.username,
+            count=self.failure_count,
+            date=date_str
+        )
+
+    def __str__(self):
+        """Str -> Username: count - date."""
+        date_str = '-'
+        if self.lockout_until is not None:
+            date_str = self.lockout_until.isoformat()
+
+        return '{username}: {count} - {date}'.format(
+            username=self.user.username,
+            count=self.failure_count,
+            date=date_str
+        )
```

Comment:
Usernames aren't always ASCII. Please use the `six.python_2_unicode_compatible` decorator on this class and return a unicode object.

Reply:
(This feature is disabled by default, but can be toggled on with `ENABLE_UNICODE_USERNAME`).